build-system/compile/bundles.config.extensions.json

```diff
@@ -646,9 +646,25 @@
   },
   {
     "name": "amp-stream-gallery",
-    "version": ["0.1", "1.0"],
-    "latestVersion": "1.0",
-    "options": {"hasCss": true}
+    "version": "0.1",
+    "latestVersion": "0.1",
+    "options": {
+      "hasCss": true
+    }
+  },
+  {
+    "name": "amp-stream-gallery",
+    "version": "1.0",
+    "latestVersion": "0.1",
+    "options": {
+      "hasCss": true,
+      "npm": {
+        "component.js": {
+          "preact": "component-preact.js",
+          "react": "component-react.js"
+        }
+      }
```

Comment:
```suggestion
      "npm": true
```

Reply:
Awesome, thanks!  I saw that you just merged your change this morning!